scripts/staging/mirror-images.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Mirror container images from source registry to target registry using crane.
+# Preserves multi-platform support and image digests.
+
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+ROOT_DIR="$SCRIPT_DIR/../.."
+
+source "$ROOT_DIR/mayastor/scripts/utils/log.sh"
+
+SOURCE_NAMESPACE=""
+TARGET_NAMESPACE=""
+TAG=""
+
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --source-namespace)
+            SOURCE_NAMESPACE="$2"
+            shift 2
+            ;;
+        --target-namespace)
+            TARGET_NAMESPACE="$2"
+            shift 2
+            ;;
+        --tag)
+            TAG="$2"
+            shift 2
+            ;;
+        *)
+            log_fatal "Unknown option: $1"
+            ;;
+    esac
+done
+
+if [[ -z "$SOURCE_NAMESPACE" ]] || [[ -z "$TARGET_NAMESPACE" ]] || [[ -z "$TAG" ]]; then
+    log_fatal "Usage: $0 --source-namespace <source> --target-namespace <target> --tag <tag>"
+fi
+
+IMAGES=(
+    "openebs-upgrade-job"
```

Comment:
Can we fetch this from the release.sh?

Reply:
hmm some code might be needed to fetch it from the derivation then?
or some other way to make sure this is kept up to date

Reply:
Yeah, not sure. Left it as it is for the time being.